ruby/private/sdk.bzl

```diff
@@ -1,15 +1,20 @@
 load(
-    ":host_runtime.bzl",
-    _ruby_host_runtime = "ruby_host_runtime",
+    "@bazelruby_ruby_rules//ruby/private/toolchains:ruby_runtime.bzl",
+    _ruby_runtime = "ruby_runtime",
 )
 
-def _register_host_runtime():
-    _ruby_host_runtime(name = "org_ruby_lang_ruby_host")
+def ruby_register_toolchains(version = "host"):
+    """Registersr ruby toolchains in the WORKSPACE file."""
+
+    supported_versions = ["host", "2.6.3", "2.6.5"]
+    if version in supported_versions:
+        _ruby_runtime(
+            name = "org_ruby_lang_ruby_toolchain",
+            version = version,
+        )
+    else:
+        fail("unknown ruby version in `ruby_register_toolchains`")
```

Comment:
Change this fail message to say 'unsupported ruby version' instead of unknown and include the list of supported versions in the output.